### PR TITLE
Include webserver in `admin ports --help` example

### PR DESCRIPTION
**Pull request by manics**
The help message should've been updated as part of https://github.com/openmicroscopy/openmicroscopy/pull/3331

### CODE REVIEW

File: components/tools/OmeroPy/src/omero/plugins/admin.py [hunk 1]
```diff
@@ -327,12 +327,12 @@ def __init__(this, name, help, wait=False):
...
 Examples:
 
-    %(prog)s --prefix=1                             # sets ports to: 14061, \
-14063, 14064
-    %(prog)s --prefix=1 --revert                    # sets ports back to: \
-4061, 4063, 4064
-    %(prog)s --registry=4444 --tcp=5555 --ssl=6666  # sets ports to: 4444 \
-5555  6666
+    %(prog)s --prefix=1
+        # sets ports to: 14061, 14063, 14064, 14080
```
Comment (sbesson) [inline components/tools/OmeroPy/src/omero/plugins/admin.py]: If we move these comments to separate lines, I would be in favor of unifying our policy. My suggestion would be to:
- align them with the command 
- have the comment before the command cf https://github.com/openmicroscopy/openmicroscopy/blob/develop/components/tools/OmeroPy/src/omero/plugins/import.py#L49.

Thoughts? /cc @ximenesuk

**Comment by sbesson** [comment]
One policy comment else :+1: for the content change.

**Comment by manics** [comment]
Rewritten. Do you also want ` %(prog)s` -> `bin/omero admin ports`?

**Comment by sbesson** [comment]
No this is fine. Thanks.

**Comment by manics** [comment]
--no-rebase

**Pull request merged**